docs/deprecations.md

```diff
@@ -25,3 +25,4 @@ being deprecated.
 | [Several fields of Task.Step are deprecated](https://github.com/tektoncd/pipeline/issues/4737)                                                                                                                               | v0.36.0                                                              | Beta                                                                                                   | Feb 25, 2023                        |
 | [ClusterTask is deprecated](https://github.com/tektoncd/pipeline/issues/4476)                                                                                                                                                | v0.41.0                                                              | Beta                                                                                                   | July 13, 2023                       |
 | [`pipelineRef.bundle` and `taskRef.bundle` are deprecated](https://github.com/tektoncd/pipeline/issues/5514)                                                                                                                 | v0.41.0                                                              | Alpha                                                                                                  | July 13, 2023                       |
+| [`taskrun.status.cloudEvents` are deprecated](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md))                                                                                     | v0.44.0                                                              | Alpha                                                                                                  | Oct 11, 2023                        |
```

Comment:
This is a field of v1beta1 TaskRun, so the stability level is beta

Reply:
Resolved.

Thanks Lee. I mistook it as the same for the pipelineResources but these have already been promoted to beta.